File: Modul-1/C/PRAK104-2310817320007-ZahraNabila.c
#include <stdio.h>

int main ()
{

    int hargaA;
printf("Harga sepatu A adalah = ");
scanf("%d", &hargaA);

    int hargaB;
printf("Harga sepatu B adalah = ");
scanf("%d", &hargaB);


    float hargaADiskon=0.13;
    float hargaBDiskon=0.21;

printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi = %.0f\n", hargaA - ( hargaA * hargaADiskon));
printf("Sepatu A mendapat diskon 21%% sehingga harganya menjadi = %.0f\n", hargaB - ( hargaB * hargaBDiskon));

    return 0;
}

File: Modul-1/Python/PRAK104-2310817320007-ZahraNabila.py
hargaA = int(input("Harga sepatu A adalah = "))
hargaB = int(input("Harga sepatu B adalah = "))


hargaADiskon=0.13
hargaBDiskon=0.21

print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi  {hargaA - ( hargaA * hargaADiskon ):.0f}")
print(f"Sepatu A mendapat diskon 21% sehingga harganya menjadi  {hargaB - ( hargaB * hargaBDiskon ):.0f}")
